portal-web/test/functional/com/liferay/portalweb/macros/FormPortletConfiguration.macro

```diff
@@ -2,6 +2,10 @@ definition {
 
 	var formName = "Created Form Name";
 
+	macro changeLanguageTo {
```

Comment:
Change this macro for macro
 changeLanguageTo {
		Click(
			changeLanguageTo = "${getLanguage}",
			locator1 = "FormPortletConfiguration#SELECT_LANGUAGE");
		Click(
			changeLanguageTo = "${changeLanguageTo}",
			locator1 = "FormPortletConfiguration#SELECT_LANGUAGE");
	}